pnpm-lock.yaml

```diff
@@ -24653,7 +24655,7 @@ packages:
   /axios/0.21.4_debug@4.3.2:
     resolution: {integrity: sha512-ut5vewkiu8jjGBdqpM44XxjuCjq9LAKeHVmoVfHVzy8eHgxxq8SbAVQNovDA8mVi05kP0Ea/n/UzcSHcTJQfNg==}
     dependencies:
-      follow-redirects: 1.15.2
+      follow-redirects: 1.15.2_debug@4.3.2
```

Comment:
I've seen things like this happen before and I don't know why they do... but do we want to revert this and the corresponding piece added in line 30134?

Reply:
Good catch! Thanks!